src/shared/components/palette/case-flag/components/update-flag/update-flag.component.ts

```diff
@@ -0,0 +1,71 @@
+import { Component, EventEmitter, Input, OnInit, Output } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { ErrorMessage } from '../../../../../domain';
+import { CaseFlagState, FlagDetail } from '../../domain';
+import { CaseFlagFieldState, CaseFlagWizardStepTitle, UpdateFlagErrorMessage, UpdateFlagStep } from '../../enums';
+
+@Component({
+  selector: 'ccd-update-flag',
+  templateUrl: './update-flag.component.html'
+})
+export class UpdateFlagComponent implements OnInit {
+
+  @Input() public formGroup: FormGroup;
+  @Input() public selectedFlagDetail: FlagDetail;
+
+  @Output() public caseFlagStateEmitter: EventEmitter<CaseFlagState> = new EventEmitter<CaseFlagState>();
+
+  public updateFlagTitle = '';
+  public errorMessages: ErrorMessage[] = [];
+  public updateFlagNotEnteredErrorMessage: UpdateFlagErrorMessage = null;
+  public updateFlagCharLimitErrorMessage: UpdateFlagErrorMessage = null;
+  public updateFlagHint: UpdateFlagStep;
+  public updateFlagCharLimitInfo: UpdateFlagStep;
+  public readonly updateFlagControlName = 'flagComments';
+  private readonly commentsMaxCharLimit = 100;
```

Comment:
@tarikdogan-hmcts - this value should be `200` (please check the ticket)

Reply:
done